zap/zap.gradle.kts

```diff
@@ -41,56 +41,56 @@ tasks.named<JacocoReport>("jacocoTestReport") {
 }
 
 dependencies {
-    api("com.fifesoft:rsyntaxtextarea:3.0.4")
+    api("com.fifesoft:rsyntaxtextarea:3.1.1")
     api("com.github.zafarkhaja:java-semver:0.9.0")
     api("commons-beanutils:commons-beanutils:1.9.4")
-    api("commons-codec:commons-codec:1.13")
-    api("commons-collections:commons-collections:3.2.2")
```

Comment:
This needs to be kept, unless we want to stop distribute it in core. We could start by changing it to `runtimeOnly` (or just remove it, but we should quick check that we are not breaking known add-ons).

Reply:
I guess we have to keep it for now, it seems used by ascanrules.

Reply:
Sounds good.